<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>Source Sans Pro Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1051</width>
      <height>621</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Sans Pro Light</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Menu">
     <attribute name="title">
      <string>Menu</string>
     </attribute>
     <widget class="QTableWidget" name="menuTable">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>491</width>
        <height>501</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Discription</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>10</y>
        <width>541</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Terminus</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>20</y>
        <width>321</width>
        <height>192</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Source Sans Pro Semibold</family>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Dish name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="dishName"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Source Sans Pro Semibold</family>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Dish discription</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="dishDiscription"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Source Sans Pro Semibold</family>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Dish Price</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="dishPriceSpinBox"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="addDish">
         <property name="font">
          <font>
           <family>Source Sans Pro Light</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>230</y>
        <width>321</width>
        <height>92</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_20">
           <property name="font">
            <font>
             <family>Source Sans Pro Semibold</family>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>DishName</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="menuDishNameDelete"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="delete_2">
         <property name="font">
          <font>
           <family>Source Sans Pro Light</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_14">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>320</y>
        <width>321</width>
        <height>42</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QPushButton" name="sortMenuPrices">
         <property name="font">
          <font>
           <family>Source Sans Pro Light</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="text">
          <string>Sort by price</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="popular">
         <property name="font">
          <font>
           <family>Source Sans Pro Light</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="text">
          <string>Popular!!!</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_15">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>370</y>
        <width>321</width>
        <height>42</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QPushButton" name="openFile">
         <property name="font">
          <font>
           <family>Source Sans Pro Light</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Open file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveFile">
         <property name="font">
          <font>
           <family>Source Sans Pro Light</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Save file</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Orders</string>
     </attribute>
     <widget class="QTableWidget" name="orderTable">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>591</width>
        <height>521</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>591</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>591</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <column>
       <property name="text">
        <string>Dish name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Order number</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Table number</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Quantity</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>160</y>
        <width>351</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="tableNumberLabel">
           <property name="font">
            <font>
             <family>Source Sans Pro Semibold</family>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string> Table number:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="tableNumber"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="dishAmountLabel">
           <property name="font">
            <font>
             <family>Source Sans Pro Semibold</family>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Quantity</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="dishAmount"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <family>Source Sans Pro Semibold</family>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string> Dish name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="menuBox"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="addOrder">
         <property name="font">
          <font>
           <family>Source Sans Pro Light</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Add Order</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>32</y>
        <width>351</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="font">
            <font>
             <family>Source Sans Pro Semibold</family>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Order number</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="orderNumberCheck"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>Source Sans Pro Semibold</family>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Check sum:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="checkLabel">
           <property name="font">
            <font>
             <family>Source Sans Pro Light</family>
             <pointsize>14</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="checkButton">
         <property name="font">
          <font>
           <family>Source Sans Pro Light</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Get check</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>310</y>
        <width>351</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_24">
           <property name="font">
            <font>
             <family>Source Sans Pro Semibold</family>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Order number</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="orderNumberChange"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="label_26">
           <property name="font">
            <font>
             <family>Source Sans Pro Semibold</family>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Table number</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="tableNumberChange"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_29">
           <property name="font">
            <font>
             <family>Source Sans Pro Semibold</family>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Dish name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="newDishName"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_27">
           <property name="font">
            <font>
             <family>Source Sans Pro Semibold</family>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Quantity</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="amountOrderChange"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="changeButton">
         <property name="font">
          <font>
           <family>Source Sans Pro Light</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Change</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
